RadarSDK/Radar.m

```diff
@@ -711,6 +711,53 @@ + (void)getMatrixFromOrigins:(NSArray<CLLocation *> *_Nonnull)origins
                                         }];
 }
 
++ (void)sendEvent:(NSString *)name
+     withMetadata:(NSDictionary *_Nullable)metadata
+completionHandler:(RadarSendEventCompletionHandler)completionHandler {
+
+    // TODO(jsani): check if permissions are enabled for events that can occur before location permission prompts
+    // if not, send the request with location = NULL
```

Comment:
Stick this in a Shortcut ticket so that we don't forget about it.

Reply:
I actually want to include this functionality in the PR, asked you and Nick about the best way to achieve it - https://github.com/radarlabs/radar-sdk-ios/pull/190#discussion_r906421525

we'll need some way to expose the `locationAuthorizationStatus` to this class